awx/api/generics.py

```diff
@@ -712,3 +713,31 @@ def get_queryset(self):
         for r in roles:
             ancestors.update(set(r.ancestors.all()))
         return User.objects.filter(roles__in=list(ancestors)).distinct()
+
+
+class CopyAPIView(ListAPIView):
```

Comment:
I think it would be better to inherit from some object-centric view. GenericAPIView and RetrieveAPIView are some of the obvious candidates.